tests/Resources/Common.robot

```diff
@@ -93,11 +93,23 @@ Wait Until HTTP Status Code Is
 
 Check HTTP Status Code
     [Documentation]     Verifies Status Code of URL Matches Expected Status Code
-    [Arguments]  ${link_to_check}  ${expected}=200
-    ${response}=    RequestsLibrary.GET  ${link_to_check}   expected_status=any
+    [Arguments]  ${link_to_check}    ${expected}=200    ${timeout}=20
+    ${headers}=    Create Dictionary    User-Agent=Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/102.0.0.0 Safari/537.36
+    ${response}=    RequestsLibrary.GET  ${link_to_check}   expected_status=any    headers=${headers}   timeout=${timeout}
     Run Keyword And Continue On Failure  Status Should Be  ${expected}
     [Return]  ${response.status_code}
 
+URLs HTTP Status Code Should Be Equal
+    [Documentation]    Given a list of link web elements, extracts the URLs and
+    ...                checks if the http status code expected one is equal to the
```

Comment:
Statement seems to be incomplete **checks if the http status code expected one is equal to the** you can write something like **check if the http status code return is equal to the expected one**

Reply:
it's like the RF built-in keywords: `Should Be Equal` for example

Reply:
I understand but This is not keyword name right it is the information what is the purpose of this keyword. If you ook at the documentation of **Should Be Equal** for example it explain in many level. What i am trying to say is it is not keyword it the documentation and i believe it is not good idea to treat the statement  like a keyword

Reply:
anyway, I've changed to `URLs HTTP Status Code Should Be Equal To`. Sounds good?

Reply:
it's okay